src/lib.rs

```diff
@@ -0,0 +1,28 @@
+extern crate atom_syndication;
+extern crate chrono;
+extern crate json;
+#[macro_use]
+extern crate lazy_static;
+extern crate rand;
+extern crate reqwest;
+#[macro_use]
+extern crate serenity;
+
+pub mod common_funcs;
+pub mod constants;
+mod faq_system;
+mod levenshtein;
+mod linkmod;
+mod prefix_control;
+mod recipe_system;
+mod simple_commands;
+mod web_requesting;
+
```

Comment:
All commands are reexported from this module. That way in main.rs, you can just use wikibot::commands::* and reduce the need to search around.